### PR TITLE
add time_filter_re for Win11 guest

### PR DESCRIPTION
**Pull request by YanhuiMa**
ID: 2002633
Signed-off-by: yama <yama@redhat.com>

### CODE REVIEW

**Comment by YanhuiMa** [comment]
PASS 1-Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.timerdevice.boot_test.with_boot.without_host_load.clksource_unchanged.base_localtime.clock_host

File: virttest/shared/cfg/guest-os/Windows.cfg [hunk 1]
```diff
@@ -125,6 +125,8 @@
         rtc_drift = "slew"
         time_command = "echo TIME: %date% %time%"
         time_filter_re = "(?<=TIME: \w\w\w ).{19}(?=\.\d\d)"
+        Win11:
+            time_filter_re = "(?<=TIME: ).{19}(?=\.\d\d)"
```
Comment (yanan-fu) [inline virttest/shared/cfg/guest-os/Windows.cfg:129]: @YanhuiMa  If i understand correctly, this parameter is used by:
https://github.com/avocado-framework/avocado-vt/blob/3de21fc1f3b725609695545ee54f4fc514ce2b77/virttest/utils_test/__init__.py#L317

The usage is: `re.findall(time_filter_re, content)[0]`.

So, suggest use one pattern for all: `time_filter_re = 'TIME:\s+(?:\w{3}\s|)(\d{2}/\d{2}/\d{4}\s+\d{1,2}:\d{2}:\d{2})(?=\.\d{2})'`

Comment (YanhuiMa) [reply virttest/shared/cfg/guest-os/Windows.cfg]: Yes, it works. And time_format also needs to been modified. I have added it for Win11

**Comment by YanhuiMa** [comment]
1-Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.timerdevice.boot_test.with_boot.without_host_load.clksource_unchanged.base_localtime.clock_host.q35 Finshed 2021-09-15 23:50:28 2021-09-15 23:52:50 PASS 142.100269
2-Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.timerdevice.boot_test.with_boot.without_host_load.clksource_unchanged.base_localtime.clock_host.q35 Finshed 2021-09-15 23:52:50 2021-09-15 23:55:32 PASS 161.826289

**Comment by yanan-fu** [comment]
@YanhuiMa  It is not a good idea to change time_format for different version.
The time_format in the output of time_command can be configured by the `regional format`.    Neither `time_format` nor `time_filter_re` need be updated if choose `English(United State)`.

The fix should be in the unattended file.

**Pull request closed without merging**